scripts/us_census/acs5yr/subject_tables/subject_table_test.py

```diff
@@ -133,6 +142,6 @@ def test_csv_mcf_column_map(self):
                     self.assertEqual(test_cmap, expected_cmap)
 
                     # Test CSV
-                    test_csv, expected_csv = _read_files(
+                    test_df, expected_df = _read_csv_files(
                         test_csv_path, paths['csv'])
-                    self.assertEqual(test_csv, expected_csv)
+                    assert_frame_equal(test_df, expected_df)
```

Comment:
nice find. Can we also set check_less_precise=True so it checks for 3 digit precision.